src/api/utils.ts

```diff
@@ -1,28 +1,30 @@
 import Axios, { AxiosError } from 'axios';
 import _ from 'lodash';
+import { captureException } from '../sentry';
 
 type OnError = <T>(arg0: AxiosError<T>) => void;
 
 export const handleApiError = <T>(error: AxiosError<T>, onError?: OnError) => {
   if (Axios.isCancel(error)) {
-    console.error('Request canceled:', error.message);
+    captureException(error, 'Request canceled');
   } else {
+    let message = `Error config: ${error.config}\n`;
     if (error.response) {
       // The request was made and the server responded with a status code
       // that falls out of the range of 2xx
-      console.error('Response data:', error.response.data);
-      console.error('Response status:', error.response.status);
-      console.error('Response headers:', error.response.headers);
+      message += `Response data: ${error.response.data}\n`;
+      message += `Response status: ${error.response.status}\n`;
```

Comment:
Can we add these parameters as more structured data instead of a string?
Just thinking about the ability to search easily.

I'm not familiar to sentry and what it offers

Reply:
Yeah, Im not sure how it is going to look like. Im still waiting for access to Sentry. Im sure we will tweak stuff once we can see how it looks in Sentry UI